civitai-downloader-v2/tests/unit/test_error_handling.py

```
@@ -0,0 +1,292 @@
+#!/usr/bin/env python3
+"""
+Unified error handling system tests.
+Tests for comprehensive error handling with fallback mechanisms and context tracking.
+"""
+
+import pytest
```

Ruff (0.12.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)